src/App.js

```diff
@@ -13,7 +14,9 @@ function App() {
           <Route path="/" exact element={ <Home /> }/> 
           <Route path="/Quiz" exact element={ <Quiz /> }/>
           <Route path="/Result" exact element={ <Result /> }/>
-          <Route path="/Guide" exact element={ <Guide /> }/>
+          <Route path="/guideline"  element={ <Guide /> }>
+            <Route path=":guidelineId" element = {<PlayNowQuiz/>}/>
```

Comment:
Add a 404 not found page. It will better

Reply:
I will add, thanks for reminding .